src/app/import/utils/subscription-details/validation.ts

```diff
@@ -130,17 +130,24 @@ export const assertSubscriptionDetailEditable: SubscriptionDetailValidationFn =
 export const assertSubscriptionDetailType: SubscriptionDetailValidationFn = (
   entry,
 ) => {
+  const ALLOWED_VSS_TYPES = [
+    SubscriptionDetailType.Int,
+    SubscriptionDetailType.Currency,
+    SubscriptionDetailType.ShortText,
+    SubscriptionDetailType.Text,
+  ];
```

Comment:
@caebr Dies war in der alten Implementation Teil des Interneteingabe-Checks. Ich habe das aber in den Type Check reingenommen.